code/modules/lighting/lighting_object.dm

```diff
@@ -14,6 +14,9 @@
 	var/needs_update = FALSE
 	var/turf/myturf
 
+	/// Cached value of max generated during [update()][/atom/movable/lighting_object/proc/update]
+	var/tmp/cached_max
```

Comment:
Why is it a `tmp` var? We don't really plan to add lighting objects to save files, do we?

Reply:
"The tmp type modifier indicates that an object variable should not be automatically written to the save file. **This could mean that the variable is transient—that is, it is calculated at run-time and need not be saved.** It could also indicated that the designer will handle saving of that variable specially and wishes to bypass the automated routine. "

Reply:
That's like 99.99% of the vars out there.
No need for this.